Quadriga/src/main/java/edu/asu/spring/quadriga/rest/DictionaryRestController.java

```diff
@@ -213,6 +220,37 @@ public String listWorkspaceDictionaries(@PathVariable("workspaceId") String work
 	
 	}
 	
+	@AccessPolicies({ @ElementAccessPolicy(type = CheckedElementType.WORKSPACE,paramIndex = 2, userRole = {RoleNames.ROLE_WORKSPACE_COLLABORATOR_ADMIN} )})

```

Comment:
users with role "ROLE_WORKSPACE_COLLABORATOR_CONTRIBUTOR" should also be able to see dictionaries.